api/controllers/v1/slack.ts

```diff
@@ -193,110 +188,42 @@ export const DeployRepo = async (event: any = {}, context: any = {}): Promise<an
       if (isStableBranch) {
         stable = '-g';
       }
+
+      newPayload.stable = stable;
+      newPayload.aliased = true;
+      newPayload.primaryAlias = true;
+
       // we use the primary alias for indexing search, not the original branch name (ie 'master'), for aliased repos
       if (urlSlug) {
-        newPayload = createPayload(
-          'productionDeploy',
-          repoOwner,
-          repoName,
-          branchName,
-          hashOption,
-          repoInfo.project,
-          repoInfo.prefix[c.get<string>('env')],
-          urlSlug,
-          true,
-          true,
-          stable
-        );
-        await deployRepo(
-          createJob(newPayload, jobTitle, jobUserName, jobUserEmail),
-          consoleLogger,
-          jobRepository,
-          parallelJobRepo,
-          parallelUrl,
-          parallelPrefix
-        );
+        newPayload.urlSlug = urlSlug;
+        console.log('Present URLSLUG', urlSlug, newPayload);
+        depolayable.push(createJob(Object.assign({}, newPayload), jobTitle, jobUserName, jobUserEmail));
         jobCount += 1;
       }
       if (non_versioned) {
-        newPayload = createPayload(
-          'productionDeploy',
-          repoOwner,
-          repoName,
-          branchName,
-          hashOption,
-          repoInfo.project,
-          repoInfo.prefix[c.get<string>('env')],
-          '',
-          true,
-          true,
-          stable
-        );
-        await deployRepo(
-          createJob(newPayload, jobTitle, jobUserName, jobUserEmail),
-          consoleLogger,
-          jobRepository,
-          parallelJobRepo,
-          parallelUrl,
-          parallelPrefix
-        );
+        newPayload.urlSlug = '';
```

Comment:
This assignment may already be covered when we first initialize `newPayload`:
```
non_versioned ? '' : branchName,
```
Should we remove this additional assignment or should we keep it here for transparency?

Reply:
removed